insights/parsers/mount.py

```diff
@@ -428,24 +428,31 @@ def _parse_mount_options(mount_options):
     opts = dict()
     sp_opts = mount_options.split(',')
     start_ndx = 0
+    in_quote = False
     for i, opt in enumerate(sp_opts):
-        # Look for option="... start of quoted value
-        if '="' in opt:
+        # Look for option="... start of quoted value, and no end quote
+        if '="' in opt and not opt.endswith('"'):
             start_ndx = i
+            in_quote = True
 
         # Look for closing quote of option="..." and if found recombine value
-        elif '"' in opt:
+        elif in_quote and opt.endswith('"'):
             last_ndx = i
             comb_opt = ','.join(sp_opts[start_ndx:last_ndx + 1])
             opt_name, opt_value = comb_opt.split('=', 1)
             # Remove leading and trailing quotes
             opts[opt_name] = opt_value[1:-1]
+            in_quote = False
 
         # Else just a normal option or option=value
         else:
             if '=' in opt:
                 opt_name, opt_value = opt.split('=', 1)
-                opts[opt_name] = opt_value
+                # Check for quoted values and remove quotes if present
+                if not (opt_value.startswith('"') and opt_value.endswith('"')):
```

Comment:
@bfahr - I think you may have tried thinking of that, but I just want to ask: can the `opt_value.strip('"')` be used here?

Reply:
@xiangce yes that would work, thanks for pointing that out.